robocopkaka/bootcamp-29-project
# #158256702 Show details of featured centers properly

#### What does this PR do?
display details of featured centers properly on the home page
#### Description of Task to be completed?
* remove home-container class from `Home` component
* move sideNav initialization to `Header` component
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install` to install all dependecies
* run `npm run start:all` to start up the application
* go to `localhost:8000` on your browser
* details of the featured centers should be properly  displayed
* click on `Centers` on the navbar
* switch to mobile view
* click on the hamburger menu at the top of the page
* a side navbar should be opened
#### Any background context you want to provide?
* a featured center's details wasn't properly displayed on the home page prior to this PR
* sidenav only worked on the home page
#### What are the relevant pivotal tracker stories?(if applicable)
#158256702
